src/main/templates/contenttypes.hbs

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {{>header}}
+  <!-- Main content starts - do not edit above this line -->
+    <table id="sorttable" class="table table-small table-hover display">
+      <thead class="thead-light">
+        <tr>
+          <th scope="col" class="d-none">Sort Order</th>
+          <th scope="col" class="align-text-top">DCNC Code <small>(For CTT)</small></th>
+          <th scope="col" class="align-text-top" width="30%">Description</th>
+          <th scope="col" class="align-text-top">ContentKind Value <small>(For DCP Metadata)</small></th>
```

Comment:
Suggest having a common _ContentKind (for DCP Metadata)_ parent label for both _Value_ and _Scope_ column, i.e. a [two-tier column heading](https://www.w3.org/WAI/tutorials/tables/irregular/#table-with-two-tier-headers)

Reply:
This is impossible with the sorting headers script (datatables).